src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/RoslynSettingsFileInfoTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Globalization;
+using System.IO;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Roslyn.Suppressions.SettingsFile;
+
+namespace SonarLint.VisualStudio.Roslyn.Suppressions.UnitTests
+{
+    [TestClass]
+    public class RoslynSettingsFileInfoTests
+    {
+
+        [DataRow("projectkey", "projectkey")]//Testing lower case
+        [DataRow("projectKEY", "projectkey")]//Testing upper case
+        [DataRow("III", "iii")]//Testing upper case with invariant culture
+        [DataRow("project:key", "project_key")]//Testing illegal characters
+        [TestMethod]
+        public void GetSettingsFilePath_ReturnsFilePathCorrectly(string projectKey, string expectedFileName)
+        {
+            var expectedPath = Path.Combine(RoslynSettingsFileInfo.Directory, expectedFileName + ".json");
+
+            //This is to make sure normalising the keys done correctly with invariant culture
+            //https://en.wikipedia.org/wiki/Dotted_and_dotless_I 
+            using (new TemporaryCultureSwitch(new CultureInfo("tr-TR")))
```

Comment:
You could use the version of `using` without braces e.g.

`using var tempCulture = new TemporaryCultureSwitch(new CultureInfo("tr-TR"));`

I don't think it makes much difference in very short methods like this one, but it does match methods with more content inside the scope more readable.